app/src/services/realitio.ts

```diff
@@ -14,43 +16,8 @@ import { Question, QuestionLog, TransactionStep } from '../util/types'
 
 const logger = getLogger('Services::Realitio')
 
-export const realitioAbi = [
-  'function askQuestion(uint256 template_id, string question, address arbitrator, uint32 timeout, uint32 opening_ts, uint256 nonce) public payable returns (bytes32)',
-  'event LogNewAnswer(bytes32 answer, bytes32 indexed question_id, bytes32 history_hash, address indexed user, uint256 bond, uint256 ts, bool is_commitment)',
-  'event LogNewQuestion(bytes32 indexed question_id, address indexed user, uint256 template_id, string question, bytes32 indexed content_hash, address arbitrator, uint32 timeout, uint32 opening_ts, uint256 nonce, uint256 created)',
-  'function isFinalized(bytes32 question_id) view public returns (bool)',
-  'function resultFor(bytes32 question_id) external view returns (bytes32)',
-  'function submitAnswer(bytes32 question_id, bytes32 answer, uint256 max_previous)',
-  'function withdraw()',
-  'function claimWinnings(bytes32 question_id, bytes32[] history_hashes, address[] addrs, uint256[] bonds, bytes32[] answers)',
-  'function questions(bytes32 question_id) view public returns (bytes32 content_hash, address arbitrator, uint32 opening_ts, uint32 timeout, uint32 finalize_ts, bool is_pending_arbitration, uint256 bounty, bytes32 best_answer, bytes32 history_hash, uint256 bond)',
-  'event LogClaim(bytes32 indexed question_id, address indexed user, uint256 amount)',
-]
-const realitioCallAbi = [
+const askQuestionConstant = [
```

Comment:
@hexyls @KadenZipfel Hey guys so this is the only hardcoded abi left standing because its the same as the one imported from relitio.json but instead of being payable its constant...And hes not on official realitio abi so in my opinion it makes sense to leave it like this?

Reply:
Seems chill to leave it imo